demos/interpretability/README.md

```diff
@@ -0,0 +1,23 @@
+# Interpretability of node classification results using StellarGraph
+
+[StellarGraph](https://github.com/stellargraph/stellargraph) has support for inspecting several different algorithms for [node classification][nc] to understand or interpret how they came to a decision. This folder contains demos of all of them to explain how they work and how to use them as part of a TensorFlow Keras data science workflow.
+
+Interpreting a model involves training and making predictions for a model, and then analysing the predictions and the model to find which neighbours and which features had the largest influence on the prediction.
+
+## Find algorithms and demos
+
+This table lists interpretability demos, including the algorithms used.
+
+| demo | algorithm(s) |
+|---|---|
+| [GCN (dense)][gcn] | GCN, Integrated Gradients |
```

Comment:
This line doesn't render correctly for me?
<img width="392" alt="Screen Shot 2020-04-07 at 3 30 13 pm" src="https://user-images.githubusercontent.com/4321744/78633342-b2735180-78e4-11ea-821b-cb2b00fee00b.png">

Reply:
Oops, I updated the link names without fixing this.